IPython/terminal/interactiveshell.py

```diff
@@ -142,6 +142,13 @@ def refresh_style(self):
         help="Override highlighting format for specific tokens"
     ).tag(config=True)
 
+    true_color = Bool(False,
+        help=("Use 24bit colors instead of 256 colors in prompt highlighting. "
+              "If your terminal supports true color, the following command "
+              "should print 'TRUECOLOR' in orange: "
+              "printf \"\\x1b[38;2;255;100;0mTRUECOLOR\\x1b[0m\\n\"")
```

Comment:
No, that's a bash command :-)

On Tue, Jul 12, 2016 at 9:10 AM, Doug Blank <notifications@github.com>
wrote:

> In IPython/terminal/interactiveshell.py
> <https://github.com/ipython/ipython/pull/9736#discussion_r70445461>:
>
> > @@ -142,6 +142,13 @@ def refresh_style(self):
> >          help="Override highlighting format for specific tokens"
> >      ).tag(config=True)
> >
> > +    true_color = Bool(False,
> > +        help=("Use 24bit colors instead of 256 colors in prompt highlighting. "
> > +              "If your terminal supports true color, the following command "
> > +              "should print 'TRUECOLOR' in orange: "
> > +              "printf \"\\x1b[38;2;255;100;0mTRUECOLOR\\x1b[0m\\n\"")
>
> Should that be print(...) ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ipython/ipython/pull/9736/files/7304bca3a12c141bba350addd435ec9d3d0e607e#r70445461>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAUez8m342vTnu_N6Oai946CMJID10wuks5qU6BGgaJpZM4JJ5SV>
> .
>